ui_advanced_config.py

```diff
@@ -82,6 +82,14 @@ def validate_string_entry(self, input):
         else:
```

Comment:
max_turn rate is not being propagated when an engine is selected. this leads to a warning popping up when switching away from this component.
<img width="370" alt="Screen Shot 2024-04-02 at 3 06 55 PM" src="https://github.com/Leading-AI-Development-Solutions/MAIA/assets/100381732/017ce8b0-b79a-4fa6-9308-ec55c58d3762">